src/uct/ib/base/ib_iface.h

```diff
@@ -195,6 +195,7 @@ struct uct_ib_iface {
         uint8_t             enable_res_domain;   /* Disable multiple resource domains */
         uint8_t             qp_type;
         size_t              max_iov;             /* Maximum buffers in IOV array */
+        uint8_t             is_global;           /* Force use GRH in address */
```

Comment:
can we avoid adding extra field? i think it makes it even more confusing
need to somehow unite config.is_global and is_global_addr